steps/update-jira-release-status/0.0.1/step.yml

```diff
@@ -0,0 +1,89 @@
+title: Jira Release Status Update
```

Comment:
Can you please change it to `Update Jira Relase Status` in the next release so it follows the dev guidelines (title shoulr be Verb + Subject. See https://github.com/bitrise-io/bitrise-steplib/pull/4083/files#r1474044575).
Will accept the Step conditionally on this for now.